packages/es-components/src/components/containers/tabPanels/TabPanel.md

```diff
@@ -0,0 +1,30 @@
+Tab Panels allow us to store lots more data on the screen and only show the amount we want, when we want. This works by having a Tab contain context, or a Tab List that contains elements. Each direct child of the tab list mush contain optiontext that will turn into the title in the dropdown. Tabs are able to have a full node inside instead of just text. This allows us to put in icons and other styling.
+
+```
+<div>
+<TabPanel optionKeyFunc={((x) => x.replace(/\s/g, ''))}>
+  <TabPanel.Tab name="Hi there">
+    <p>
+      HELLO WORLD!!!!
+    </p>
+  </TabPanel.Tab>
+  <TabPanel.Tab name={<div><Icon name="certificate" /><span>Interesting</span><p>5 plans</p></div>}>
```

Comment:
This would look better simpler, for example just: <div><Icon name="certificate" /> Interesting 5 plans</div>